Installation issue: blast-plus (missing cpio)

### Steps to reproduce the issue

```console
cat > spack.yaml << EOF
spack:
  view: true
  specs:
  - blast-plus@2.12.0
  container:
    extra_instructions:
      final: RUN (echo /opt/view/lib && echo /opt/view/lib64) > /etc/ld.so.conf.d/spack-view.conf
        && ldconfig
    images:
      os: ubuntu:22.04
      spack:
        ref: develop
EOF
spack containerize > Dockerfile
docker build -t blast:2.12.0 .
```


### Error message

```
 59307    done
     59308    /bin/bash: line 3: cpio: command not found
     59309    /bin/bash: line 3: cpio: command not found
  >> 59310    make[1]: *** [Makefile:47: install-toolkit] Error 127
     59311    make[1]: Leaving directory '/tmp/root/spack-stage/spack-stage-bla
              st-plus-2.12.0-rlog3ww57onydmbgmpbqj25nvcmayses/spack-src/c++'
  >> 59312    make: *** [Makefile:63: install] Error 2
```

### Information on your system

c7g-4x Instance

### Additional information

_No response_

### General information

- [X] I have run `spack debug report` and reported the version of Spack/Python/Platform
- [X] I have run `spack maintainers <name-of-the-package>` and **@mentioned** any maintainers
- [X] I have uploaded the build log and environment files
- [X] I have searched the issues of this repo and believe this is not a duplicate

--- a/var/spack/repos/builtin/packages/blast-plus/package.py
+++ b/var/spack/repos/builtin/packages/blast-plus/package.py
@@ -10,10 +10,11 @@ class BlastPlus(AutotoolsPackage):
     """Basic Local Alignment Search Tool."""
 
     homepage = "https://blast.ncbi.nlm.nih.gov/"
-    url = "https://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/2.12.0/ncbi-blast-2.12.0+-src.tar.gz"
+    url = "https://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/2.13.0/ncbi-blast-2.13.0+-src.tar.gz"
 
     maintainers = ["weijianwen"]
 
+    version("2.13.0", sha256="89553714d133daf28c477f83d333794b3c62e4148408c072a1b4620e5ec4feb2")
     version("2.12.0", sha256="fda3c9c9d488cad6c1880a98a236d842bcf3610e3e702af61f7a48cf0a714b88")
     version("2.11.0", sha256="d88e1858ae7ce553545a795a2120e657a799a6d334f2a07ef0330cc3e74e1954")
     version("2.9.0", sha256="a390cc2d7a09422759fc178db84de9def822cbe485916bbb2ec0d215dacdc257")

--- a/var/spack/repos/builtin/packages/blast-plus/package.py
+++ b/var/spack/repos/builtin/packages/blast-plus/package.py
@@ -46,7 +46,7 @@ class BlastPlus(AutotoolsPackage):
     # No...
     # depends_on :mysql => :optional
 
-    depends_on('cpio', type='build')
+    depends_on("cpio", type="build")
 
     variant("static", default=False, description="Build with static linkage")
     variant("jpeg", default=True, description="Build with jpeg support")

--- a/var/spack/repos/builtin/packages/blast-plus/package.py
+++ b/var/spack/repos/builtin/packages/blast-plus/package.py
@@ -48,7 +48,6 @@ class BlastPlus(AutotoolsPackage):
 
     depends_on('cpio', type='build')
 
-
     variant("static", default=False, description="Build with static linkage")
     variant("jpeg", default=True, description="Build with jpeg support")
     variant("png", default=True, description="Build with png support")

--- a/var/spack/repos/builtin/packages/blast-plus/package.py
+++ b/var/spack/repos/builtin/packages/blast-plus/package.py
@@ -46,6 +46,9 @@ class BlastPlus(AutotoolsPackage):
     # No...
     # depends_on :mysql => :optional
 
+    depends_on('cpio', type='build')
+
+
     variant("static", default=False, description="Build with static linkage")
     variant("jpeg", default=True, description="Build with jpeg support")
     variant("png", default=True, description="Build with png support")

